### PR TITLE
Add adapter for Brightcom

### PR DESCRIPTION
**Pull request by matan-ybrant**

### CODE REVIEW

**Comment by mkendall07** [comment]
Hi @matan-ybrant 
Thanks for the PR. Can you please remove the `build/*` artifacts from the PR and also add a usage example for your adapter to this file: https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/pbjs_example_gpt.html

**Comment by matan-ybrant** [comment]
Hi @mkendall07 
Thanks for your response!
I've removed the build/* files, as requested, and I'll update the example soon.

**Comment by matan-ybrant** [comment]
Hi @mkendall07 
Everything is updated as requested.
Please let me know if anything else is required.

**Comment by mkendall07** [comment]
@matan-ybrant 
Sorry for the delay, please provide test params for your bidder so we can verify end to end functionality for your adapter.

**Comment by matan-ybrant** [comment]
@mkendall07 
No problem.
Can you please tell me where you're testing from (I need to define geo settings in the server in order for you to get a bid)

**Comment by mkendall07** [comment]
We have several different locations, Portland, Philadelphia, NYC. Thanks

**Comment by mkendall07** [comment]
@matan-ybrant 
Any update here?

**Comment by matan-ybrant** [comment]
Hi @mkendall07 
Sorry for the late response.
I removed the conflicts.

To test the adapter, all you need is the param "tagId", with the value 16577 (you'll receive bids from US and Canada, for the following sizes: 160x600, 300x250, 300x600, 728x90).
The example page (pbjs_example_gpt.html) contains the working example.

**Comment by protonate** [comment]
Thanks you @matan-ybrant, I'm able to get bids back using the Brightcom adapter. Please rebase this branch onto current master, apply JSCS and jshint code style (you can incorporate the .jscs and .jshint specifications into your IDE) and flatten commits. Then we'll get this merged in.

File: src/adapters/brightcom.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+        // Add timeout data, if available
+        var PREBID_TIMEOUT = PREBID_TIMEOUT || 0;
+        var curTimeout = PREBID_TIMEOUT;
+        if (curTimeout > 0) {
+            brightcomBidReq.tmax = curTimeout;
+        }
+
+        // Define the bid request call URL
+        var bidRequestCallUrl = '//' + brightcomUrl +
```
Comment (protonate) [inline src/adapters/brightcom.js]: We are standardizing HTTPS for all adapters so please update any relative protocol code to always use HTTPS.

**Comment by matan-ybrant** [comment]
Hi @protonate,
Regarding the JSCS and JSHint styling - I ran the 'quality' task via Gulp, and it didn't throw any errors on the current version.
Did you spot any problem?

**Comment by protonate** [comment]
Ah, the reporter setting was missing from the `jscs` gulp task. You can add that back in as shown [here](https://github.com/prebid/Prebid.js/pull/273/files#diff-b9e12334e9eafd8341a6107dd98510c9R128). That change is in PR #273 and will be merged to master soon, making it the default.

Most errors are for indentation (2 spaces), no multiple var declarations and no trailing whitespace.

Thank you for pointing out the jscs reporting issue.

**Comment by matan-ybrant** [comment]
@protonate - Thanks for the information. I applied the JSCS styling, and will rebase soon.

**Comment by matan-ybrant** [comment]
Hi @protonate 
I've created a new pull request #283 
Lets continue there.
Thanks!

**Pull request closed without merging**